app/mailers/application_mailer.rb

```diff
@@ -1,4 +0,0 @@
-class ApplicationMailer < ActionMailer::Base
```

Comment:
I'm not sure we should remove this here. We no longer need to send emails in relation to user accounts but removing wider support for ActionMailer is something we should consider separately in the context of the whole application, not authentication.

Reply:
I'm not saying we can never send email, just that we don't need to right now. We can always add it properly (or from the history) if and when we need to.

Reply:
Please leave it in now.